ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -183,7 +183,6 @@ presubmits:
           value: e2e-aws-fips
         - name: TEST_COMMAND
           value: |
-            fips_check
```

Comment:
I see no need for `pull-ci-openshift-installer-master-e2e-aws-fips` and similar jobs if we feel that the MCO test already covers the FIPS angle sufficiently.  Can we just drop the jobs, instead of editing them to be identical to vanilla e2e?  Or do we feel like we still need coverage on installer PRs of installs with FIPS enabled?

Reply:
The job injects the `fips: true` install config still and we're still testing that.